explorer/app/components/ShowMore/ShowMore.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button } from "@mui/material";
+import React, { useState } from "react";
+import { Container, Wrapper } from "./ShowMore.styles";
+
+interface ShowMoreProps {
```

Comment:
@MillenniumFalconMechanic The ShowMore component is a little different from what you suggested on the issue. My thinking was that passing two props (one for the components to show and to other for the components that will be hidden) might be a little more complex to control for multiple screen sizes. So I thought that it might be better to control it using the wrapper component's height. Let me know what you think!